---
name: Aaron S. Meyer
url: http://asmlab.org
email: ameyer@ucla.edu
phone: (310) 794-4821
addrOne: 4121G Engineering V
addrTwo: Los Angeles, CA 90095
---

# Education

_Ph.D._, Biological Engineering  \hfill April 2014  
Massachusetts Institute of Technology (MIT), Cambridge, MA  
Thesis: Quantitative approaches to understanding signaling regulation of 3D cell migration
 
_B.S._, Bioengineering, magna cum laude \hfill June 2009  
University of California, Los Angeles (UCLA), CA

# Professional Experience

_Associate Professor_ \hfill 2023 – Present  
_Assistant Professor_ \hfill 2017 – 2023  
Bioengineering Department, UCLA  
Bioinformatics Interdepartmental Graduate Program, UCLA  
Computational & Systems Biology Interdepartmental Program, UCLA

_Principal Investigator & Research Fellow_ \hfill 2014 – 2017  
Koch Cancer Institute, MIT, Cambridge, MA

_Graduate Researcher in the labs of Douglas Lauffenburger & Frank Gertler_ \hfill 2009 – 2014  
Department of Biological Engineering & Koch Cancer Institute, MIT, Cambridge, MA

_Undergraduate Researcher in the lab of Daniel Kamei_ \hfill 2006 – 2009  
Department of Bioengineering, UCLA

# Submitted Publications Under Review/Revision

\leftskip 0.1in
\parindent -0.1in

46\. **Tan, Z.C.**, **A.S. Meyer**. "The structure is the message: preserving experimental context through tensor decomposition." _arXiv preprint_.

\parskip 0.1in

45\. Kojima, H., T. Morinelli, Y. Wang, **J. Chin**, **A.S. Meyer**, M. Kuo, K. Kadono, S. Yao, T. Torgerson, K. Dery, A. Bhat, E. Reed, F. Kaldas, D. Windt, D. Farmer, J. Kupiec-Weglinski, Y. Zhai. "Group 1 innate lymphoid cells protect liver transplants from ischemia reperfusion injury via IFN-γ-mediated pathway." _Submitted._

44\. **Orcutt-Jahns, B.**, J. Rodrigues Lima Junior, R.C. Rockne, A. Matache, S. Branciamore, E. Hung, A.S. Rodin, P.P. Lee, **A.S. Meyer**. "Systems profiling reveals recurrently dysregulated cytokine signaling responses in ER+ breast cancer patients’ blood." _Submitted, [bioRxiv preprint](https://www.biorxiv.org/content/10.1101/2023.10.31.564987v1)._

43\. **Creixell, M.**, **S.D. Taylor**, J. Gerritsen, **S.Y. Bae**, M. Jiang, T. Augustin, **M. Loui**, **C. Boixo**, P. Creixell, F.M. White, **A.S. Meyer**. "Dissecting signaling regulators driving AXL-mediated bypass resistance and associated phenotypes by phosphosite perturbations." _Submitted, [bioRxiv preprint](https://www.biorxiv.org/content/10.1101/2023.10.20.563266v1)._

# Refereed Publications

\leftskip 0.1in
\parindent -0.1in

42\. **Chin, J.L.**, **Z.C. Tan**, L.C. Chan, F. Ruffin, R. Parmar, R. Ahn, **S. Taylor**, A.S. Bayer, A. Hoffmann, J. Vance G Fowler, E.F. Reed, M.R. Yeaman, **A.S. Meyer**, MRSA Systems Immunobiology Group. ["Tensor modeling of MRSA bacteremia cytokine and transcriptional patterns reveals coordinated, outcome-associated immunological programs."](https://doi.org/10.1093/pnasnexus/pgae185) _PNAS Nexus._ 2024 May 4; 3(5): pgae185.

41\. **Orcutt-Jahns, B.**, **P.C. Emmel**, **E.M. Snyder**, **S.D. Taylor**, **Aaron S. Meyer**. ["Multivalent, asymmetric IL-2–Fc fusions show enhanced selectivity for regulatory T cells."](https://www.science.org/doi/full/10.1126/scisignal.adg0699) _Science Signaling._ 2023 October 17; 16(807): eadg0699.

\parskip 0.1in

40\. Peyton, S.R., L.W. Chow, S.D. Finley, A.N. Ford Versypt, R. Hill, M.L. Kemp, E.M. Langer, A.P. McGuigan, **A.S. Meyer**, S.K. Seidlits, K. Roy, S.M. Mumenthaler. ["Synthetic living materials in cancer biology."](https://www.nature.com/articles/s44222-023-00105-w) _Nature Reviews Bioengineering._ 2023 Oct 2; 1, 972–988.

39\. Terry, A.Q., H. Kojima, R.A. Sosa, F.M. Kaldas, **J.L. Chin**, Y. Zheng, B.V. Naini, D. Noguchi, J.N.- Mejia, Y.-P. Jin, R.W. Busuttil, **A.S. Meyer**, D.W. Gjertson, J.W. Kupiec-Weglinski, E.F. Reed. ["Disulfide-HMGB1 Signals Through TLR4 and TLR9 to Induce Inflammatory Macrophages Capable of Innate-Adaptive Crosstalk in Human Liver Transplantation."](https://www.amjtransplant.org/article/S1600-6135(23)00625-1/fulltext) _Am J Transplant._ 2023 Aug 9; S1600-6135(23)00625-1.

38\. **Tan, Z.C.**, A. Lux, M. Biburger, P. Varghese, **S. Lees**, F. Nimmerjahn, **A.S. Meyer**. ["Mixed IgG Fc immune complexes exhibit blended binding profiles and refine FcR affinity estimates."](https://doi.org/10.1016/j.celrep.2023.112734) _Cell Reports._ 2023 July 25; 42(7): 112734.

37\. S.M. Gross, **F. Mohammadi**, C. Sanchez-Aguila, P.J. Zhan, **A.S. Meyer**, L.M. Heiser. ["Analysis and modeling of cancer drug responses using cell cycle phase-specific rate effects."](https://www.nature.com/articles/s41467-023-39122-z) _Nature Communications._ 2023 June 10; 14: 3450.

36\. Yang, H., U.Y. Ulge, A. Quijano-Rubio, Z.J. Bernstein, D.R. Maestas, J.-H. Chun, W. Wang, J.-X. Lin, K.M. Jude, S. Singh, **B.T. Orcutt-Jahns**, P. Li, J. Mou, L. Chung, Y.-H. Kuo, Y.H. Ali, **A.S. Meyer**, W.L. Grayson, N.M. Heller, K.C. Garcia, W.J. Leonard, D.-A. Silva, J.H. Elisseeff, D. Baker, J.B. Spangler. ["Design of cell-type-specific hyperstable IL-4 mimetics via modular de novo scaffolds."](https://doi.org/10.1038/s41589-023-01313-6) _Nature Chemical Biology_. 2023 April 6. 1552–4469.

35\. **J.L. Chin**, L.C. Chan, M.R. Yeaman, **A.S. Meyer**. ["Tensor-based insights into systems immunity and infectious disease."](https://doi.org/10.1016/j.it.2023.03.003) _Trends in Immunology_. 2023 May; 44(5): 329–332.

34\. **Wilder, C.**, D. Lefaudeux, R. Mathenge, K. Kishimoto, A.Z. Munoz, M.A. Nguyen, **A.S. Meyer**, Q.J. Cheng, A. Hoffmann. ["A stimulus-contingent positive feedback loop enables IFN-β dose-dependent activation of pro-inflammatory genes."](https://doi.org/10.15252/msb.202211294) _Molecular Systems Biology_. 2023 March 17; 19: e11294.

33\. P. Kulkarni, H.S. Wiley, H. Levine, H. Sauro, A. Anderson, S.T.C. Wong, **A.S. Meyer**, P. Iyengar, K. Corlette, K. Swanson, A. Mohanty, S. Bhattacharya, A. Patel, V. Jain, R. Salgia. ["Addressing the genetic/nongenetic duality in cancer with systems biology."](https://doi.org/10.1016/j.trecan.2022.12.004) _Trends in Cancer_. 9(3), 2023 March, 185–187.

32\. Kim, H., A. Wirasaputra, **F. Mohammadi**, A.N. Kundu, J.A.E. Esteves, L.M. Heiser, **A.S. Meyer**, S.R. Peyton. ["Live Cell Lineage Tracing of Dormant Cancer Cells."](https://doi.org/10.1002/adhm.202202275) _Advanced Healthcare Materials._ 2023 Jan 10; 12(14): 2202275.

31\. **Mohammadi, F.**, **S. Visagan**, S.M. Gross, **L. Karginov**, **JC Lagarde**, L.M. Heiser, **A.S. Meyer**. ["A lineage tree-based hidden Markov model to quantify cellular heterogeneity and plasticity."](https://doi.org/10.1038/s42003-022-04208-9) _Communications Biology_. 2022 Nov 17; 5(1): 1528.

30\. VanDyke, D., M. Iglesias, J. Tomala, A. Young, J. Bridge, J. Perry, E. Gebara, A.R. Cross, L.S. Cheung, A.G. Dykema, **B. Orcutt-Jahns**, T. Henclová, J. Golias, J. Balolong, L.M. Tomasovic, D. Funda, **A.S. Meyer**, D.M. Pardoll, J. Hester, F. Issa, C.A. Hunter, M.S. Anderson, J.A. Bluestone, G. Raimondi, J.B. Spangler. ["Engineered human cytokine/antibody fusion proteins elicit targeted expansion of regulatory T cells and confer protection against autoimmune diseases."](https://doi.org/10.1016/j.celrep.2022.111478) _Cell Reports._ 2022 Oct 18; 41(3): 111478.

29\. **Creixell, M.**, H. Kim, **F. Mohammadi**, S.R. Peyton, **A.S. Meyer**. ["Systems approaches to uncovering the contribution of environment-mediated drug resistance."](https://doi.org/10.1016/j.cossms.2022.101005) _Current Opinion in Solid State & Materials Science._ 2022 Oct; 26(5): 101005.

28\. **Creixell, M.**, **A.S. Meyer**. ["Dual data and motif clustering improves the modeling and interpretation of phosphoproteomic data."](https://doi.org/10.1016/j.crmeth.2022.100167) _Cell Reports Methods._ 2022 Feb 28; 2(2): 100167.

27\. Majumder, A., S. Hosseinian, M.J. Stroud, E. Adhikari, J.J. Saller, D.M.A. Smith, D.G. Zhang, S. Agarwal, **M. Creixell**, B.S. Meyer, M.F. Kinose, K.S. Bowers, B. Fang, P.A. Stewart, E.A. Welsh, T.A. Boyle, **A.S. Meyer**, J.M. Koomen, E.B. Haura. ["Proteomic Characterization of AXL Kinase Inhibitors and Signaling Pathways."](https://mcr.aacrjournals.org/content/early/2022/01/12/1541-7786.MCR-21-0275.abstract) _Molecular Cancer Research._ 2022 Jan 7; 20(4): 542–555.

26\. **Tan, Z.C.**, **B.T. Orcutt-Jahns**, **A.S. Meyer**. ["A quantitative view of strategies to engineer cell-selective ligand binding."](https://doi.org/10.1093/intbio/zyab019) _Integrative Biology._ 2021 Nov 23; 13(11): 269–282.

25\. **Tan, Z.C.**, **A.S. Meyer**. ["A general model of multivalent binding with ligands of heterotypic subunits and multiple surface receptors."](https://doi.org/10.1016/j.mbs.2021.108714) _Mathematical Biosciences._ 2021 Dec; 342:108714.

24\. **Tan, Z.C.**, **M.C. Murphy**, **H.S. Alpay**, **S.D. Taylor**, **A.S. Meyer**. ["Tensor-structured decomposition improves systems serology analysis."](https://doi.org/10.15252/msb.202110243) _Molecular Systems Biology._ 2021 Sept 6; 17:e10243.

23\. **Farhat, A.M.**, **A.C. Weiner**, C. Posner, **Z.S. Kim**, **B. Orcutt-Jahns**, S.M. Carlson, **A.S. Meyer**. ["Modeling Cell-Specific Dynamics and Regulation of the Common Gamma Chain Cytokines."](https://doi.org/10.1016/j.celrep.2021.109044) _Cell Reports._ 2021 April 27; 35(4):109044.

22\. **Bae, S.Y.**, **N. Guan**, **R. Yan**, **K. Warner**, **S.D. Taylor**, **A.S. Meyer**. ["Measurement and models accounting for cell death capture hidden variation in compound response."](https://www.nature.com/articles/s41419-020-2462-8) _Cell Death & Disease._, 2020 Apr 20; 255(11). 

21\. Lee, C.-H., T. H. Kang, O. Godon, M. Watanabe, G. Delidakis, C.M. Gillis, D. Sterlin, D. Hardy, M. Cogné, L.E. Macdonald, A.J. Murphy, N. Tu, J. Lee, J.R. McDaniel, E. Makowski, Peter M. Tessier, **A.S. Meyer**, P. Bruhns, G. Georgiou. ["An engineered human Fc domain that behaves like a pH-toggle switch for ultra-long circulation persistence."](https://www.nature.com/articles/s41467-019-13108-2) _Nature Communications._, 2019 Nov 6; 10(1):5031.

20\. **Meyer, A.S.**, L.M. Heiser. ["Systems biology approaches to measure and model phenotypic heterogeneity in cancer."](https://doi.org/10.1016/j.coisb.2019.09.002) _Current Opinion in Systems Biology._ 2019 Oct 4; 17: 35--40.

19\. Situ, K., B.A. Chua, **S.Y. Bae**, **A.S. Meyer**, K. Morizono. ["Versatile targeting system for lentiviral vectors involving biotinylated targeting molecules."](https://doi.org/10.1016/j.virol.2018.09.017) _Virology._ 2018 Oct 2; 525: 170--181.

18\. **Robinett, R.A.**, **N. Guan**, A. Lux, M. Biburger, F. Nimmerjahn, **A.S. Meyer**. ["Dissecting FcγR Regulation Through a Multivalent Binding Model."](https://doi.org/10.1016/j.cels.2018.05.018) _Cell Systems._ 2018 Jul 25; 6(7): 1--8.

17\. Claas, A.M., L. Atta, S. Gordonov, **A.S. Meyer**, D.A. Lauffenburger. ["Systems Modeling Identifies Divergent Receptor Tyrosine Kinase Reprogramming to MAPK Pathway Inhibition."](https://link.springer.com/article/10.1007/s12195-018-0542-y) _Cellular and Molecular Bioengineering_ 2018 Jul 26; 1--19.

16\. Muffat, J., Y. Li, A. Omer, A. Durbin, I. Bosch, G. Bakiasi, **E. Richards**, **A.S. Meyer**, L. Gehrke, R. Jaenisch. ["Human iPS-derived Glial Cells and Neural Progenitors Display Divergent Responses to Zika and Dengue Infections."](https://www.pnas.org/content/115/27/7117) _Proc. Natl. Acad. Sci. U.S.A._ 2018 Jun 18; 201719266.

15\. Schwartz, A.D., L.E. Barney, L.E. Jansen, T.V. Nguyen, C.L. Hall, **A.S. Meyer**, S.R. Peyton. ["A Biomaterial Screening Approach to Reveal Microenvironmental Mechanisms of Drug Resistance."](http://pubs.rsc.org/en/content/articlelanding/2017/ib/c7ib00128b) _Integrative Biology._ 2017 Nov 14; 9: 912--924.

14\. **Zweemer, A.J.M.**, C.B. French, J. Mesfin, S. Gordonov, **A.S. Meyer**, D.A. Lauffenburger. ["Apoptotic Cell Bodies Elicit Gas6-Mediated Migration Of AXL-Expressing Tumor Cells."](https://doi.org/10.1158/1541-7786.MCR-17-0012) _Molecular Cancer Research._ 2017 Sept 18; 15 (12): 1656--1666.

13\. Archer, T.C., E.J. Fertig, S.J.C. Gosline, M. Hafner, S.K. Hughes, B.A. Joughin, **A.S. Meyer\footnote{Corresponding author.}**, S.P. Piccolo, A. Shajahan-Haq. ["Systems Approaches to Cancer Biology."](https://dx.doi.org/10.1158%2F0008-5472.CAN-16-1580) _Cancer Research._ 2016 Nov 18; 76 (23); 1--4.

12\. **Manole, S.**, **E.J. Richards**, **A.S. Meyer**. ["JNK pathway activation modulates acquired resistance to EGFR/HER2 targeted therapies."](https://aacrjournals.org/cancerres/article/76/18/5219/615800/JNK-Pathway-Activation-Modulates-Acquired) _Cancer Research._ 2016 Sept 15; 76 (18): 5219--5228.

11\. McConnell, R.E., J.E. Van Veen, M. Vidaki, A.V. Kwiatkowski, **A.S. Meyer**, D.A. Lauffenburger, F.B. Gertler. ["A Requirement for Filopodia Extension Towards Slit During Robo-Mediated Axon Repulsion."](http://jcb.rupress.org/content/213/2/261.full) _Journal of Cell Biology._ 2016 Apr 18; 213 (2): 261.

10\. Miller, M.A., M.J. Oudin, R.J. Sullivan, D.T. Frederick, **A.S. Meyer**, S. Wang, H. Im, J. Tadros, L.G. Griffith, H. Lee, R. Weissleder, K.T. Flaherty, F.B. Gertler, D.A. Lauffenburger. ["Reduced proteolytic shedding of receptor tyrosine kinases is a post-translational mechanism of kinase inhibitor resistance."](http://cancerdiscovery.aacrjournals.org/content/early/2016/03/15/2159-8290.CD-15-0933) _Cancer Discovery._ 2016 Apr; 6:331-333.

9\. Miller, M.A., M. Moss, G. Powell, R. Petrovich, L. Edwards, **A.S. Meyer**, L.G. Griffith, D.A. Lauffenburger. ["Targeting autocrine HB-EGF signaling with specific ADAM12 inhibition using recombinant ADAM12 prodomain."](http://www.ncbi.nlm.nih.gov/pubmed/26477568) _Scientific Reports._ 2015 Oct 19; 5:15150.

8\. **Meyer\footnote{Co-corresponding authors.}, A.S.**, **A.J.M. Zweemer**, D.A. Lauffenburger\footnotemark[\value{footnote}]. ["The AXL receptor is a sensor of ligand spatial heterogeneity."](http://dx.doi.org/10.1016/j.cels.2015.06.002) _Cell Systems._ 2015 Nov 29; 1(1):25-36.

7\. Riquelme, D.N., **A.S. Meyer**, M. Barzik, A. Keating, F.B. Gertler. ["Selectivity in subunit composition of Ena/VASP tetramers."](http://www.ncbi.nlm.nih.gov/pubmed/26221026) _Biosci. Rep._ 2015 Jul 28;35(5). pii: e00246.

6\. **Meyer, A.S.**, M.A. Miller, F.B. Gertler, D.A. Lauffenburger. ["The receptor AXL diversifies EGFR signaling and limits the response to EGFR-targeted inhibitors in triple-negative breast cancer cells."](http://www.ncbi.nlm.nih.gov/pubmed/23921085) _Science Signaling._ 2013 Aug 6; 6(287):ra66.

5\. Miller\footnote{Equally contributing authors.}, M.A., **A.S. Meyer**\footnotemark[\value{footnote}], M. Beste, Z. Lasisi, S. Reddy, K. Jeng, C.-H. Chen, J. Han, K. Isaacson, L.G. Griffith, D.A. Lauffenburger. ["ADAM-10 and -17 regulate endometriotic cell migration via concerted ligand and receptor shedding feedback on kinase signaling."](http://www.ncbi.nlm.nih.gov/pubmed/23674691) _Proc. Natl. Acad. Sci. U.S.A._ 2013 May 28; 110(22):E2074-83.

4\. **Meyer, A.S.**, S.K. Hughes-Alford, J.E. Kay, A. Castillo, A. Wells, F.B. Gertler, D.A. Lauffenburger. ["2D protrusion but not motility predicts growth factor-induced cancer cell migration in 3D collagen."](http://www.ncbi.nlm.nih.gov/pubmed/22665521) _Journal of Cell Biology._ 2012 Jun 11; 197(6):721-9.

3\. Kim, H.D., **A.S. Meyer**, J.P. Wagner, S.K. Alford, A. Wells, F.B. Gertler, D.A. Lauffenburger. ["Signaling network state predicts Twist-mediated effects on breast cell migration across diverse growth factor contexts."](http://www.ncbi.nlm.nih.gov/pubmed/21832255) _Mol. Cell. Proteomics._ 2011 Nov;10(11):M111.008433.

2\. **Meyer, A.S.**, R.G. Condon, G. Keil, N. Jhaveri, Z. Liu, Y.-S. Tsao. ["Fluorinert, an oxygen carrier, improves cell culture performance in deep square 96-well plates by facilitating oxygen transfer."](http://www.ncbi.nlm.nih.gov/pubmed/21954223) _Biotechnol. Prog._ 2012 Jan; 28(1):171-8.

1\. Mashayekhi, F., **A.S. Meyer**, S.A. Shiigi, V. Nguyen, D.T. Kamei. ["Concentration of mammalian genomic DNA using two-phase aqueous micellar systems."](http://www.ncbi.nlm.nih.gov/pubmed/19061237) _Biotechnol. Bioeng._ 2009 Apr 15; 102(6):1613-23.

\leftskip 0.0in
\parindent 0.0in

# Research Support & Awards

_Contact PI on all grants unless indicated otherwise._

_NSF 2404470 (Co-I)_ \hfill 2024 – 2027  
"Multi-analyte Detection Enabled by Machine Learning-Guided Voltammetry"  
Division: CHE Research and Development

_NIH NEI R01-EY011996 (Co-I)_ \hfill 2023 – 2027  
"Retinal Disease: Molecular Basis and Pathophysiology"

_NIH NIAID U19 (Co-PI)_ \hfill 2023 – 2028  
Systems Biology for Infectious Diseases Consortium  
"Systems Epigenomics of Persistent Bloodstream Infection"

_Emerging Leader Award_, Mark Foundation for Cancer Research \hfill 2023–2025  
"Tracking and Reactivating Humoral Immunity through Systems Serology"

_SEEDS Grant_ \hfill 2023 – 2024  
Merck & Co., Inc.  
"Systematic and Receptor-Specific Dissection of Fc Receptor Functions"

_NIH NIAID P01-AI120944 (Co-I)_ \hfill 2022 – 2027  
Transplant Immunology Program Project Grant  
"Innate-Adaptive Immunoregulation in Liver Transplant Ischemia/Reperfusion Injury"

_Outstanding Mentor Award_, Bruins-In-Genomics \hfill 2022

_COVID Relief Funds_, Vice Chancellor for Research Office \hfill 2022

_Milstein Abstract Award_, Cytokine Society \hfill 2021  
"Multivalency enhances the specificity of Fc-cytokine fusions"

_Northrop Grumman Excellence in Teaching Award_ \hfill 2021  

_Administrative Supplement to U01-CA215709_ \hfill 2020 – 2021  
"Mechanistic Autoencoders for Patient-Specific Phosphoproteomic Models"

_Grant_ \hfill 2020 – 2022  
Jayne Koskinas Ted Giovanis Foundation  
"Cell cycle-specific drug responses in breast cancer"

_American Cancer Society, Research Scholar Grant (co-I)_  \hfill 2020 – 2023  
"Tissue-engineered models of glioblastoma for evaluating treatment responses"

_NIH NIAID U01-AI148119_ \hfill 2019 – 2024  
Fc-Dependent Mechanisms of Antibody-Mediated Killing Consortium  
"Mapping the effector response space of antibody combinations"

_UCLA Faculty Career Development Award_ \hfill 2019 – 2020  

_UCLA Hellman Fellow_ \hfill 2019 – 2020  
"Engineering anti-tumor antibody combinations for more effective and less toxic therapies"

_Visterra, Inc. Research Agreement_ \hfill 2019 – 2021  
"IL-2 Receptor Binding Engineering"

_Administrative Supplement to U01-CA215709_ \hfill 2018 – 2019  
"Cell lineage analysis to quantify heterogeneous cell cycle responses of cancer cells"

_NIH NCI U01-CA215709_ \hfill 2017 – 2022  
Cancer Systems Biology Consortium  
"Precision Lung Cancer Therapy Design through Multiplexed Adapter Measurement"

_Fellowship Grant_ \hfill 2017 – 2019  
Terri Brodeur Breast Cancer Foundation  
"Decoding the Role of TAM Receptors _In Vivo_ Using More Specific and Potent Inhibitors"

_Finalist_, Career Awards at the Scientific Interface \hfill 2017  
Burroughs Wellcome Fund

_[Ten to Watch](http://www.amgenscholars.com/alumni/ten-to-watch)_, Amgen Scholars Foundation \hfill 2016

_AMIGOS Program Award_ \hfill 2016 – 2020  
Jayne Koskinas Ted Giovanis Foundation and Breast Cancer Research Foundation  
"Understanding the Role of Cell Plasticity in Mediating Drug Resistance"

_GPU Grant_ \hfill 2016  
NVIDIA Corporation  
"Parameterizing Stochastic Cell Signaling Pathways Through Variability Fitting"

_Frontier Research Program Initiator Award_ \hfill 2015  
Koch Institute for Integrative Cancer Research  
"Multiplexed Tools for Probing Chemokine Receptor Activation State in Breast Cancer"

_NIH Director's Early Independence Award, DP5-OD019815_  \hfill 2014 – 2019  
"Adapter-Layer RTK Signaling: Basic Understanding & Targeted Drug Resistance"  
**[Highlighted by the NIH director's office.](https://directorsblog.nih.gov/2014/10/28/creative-minds-tackling-chemotherapy-resistance/)**

_Siebel Scholar, Class of 2014_ \hfill 2013 

_Whitaker Fellowship_  \hfill 2013  
Massachusetts Institute of Technology

_Repligen Fellowship in Cancer Research_ \hfill 2012  
Koch Institute for Integrative Cancer Research

_Frontier Research Program Initiator Award_ \hfill 2011  
Koch Institute for Integrative Cancer Research  
"Global Growth Factor Reprogramming and Invasion By AXL Expression And Shedding In Breast Carcinoma"

_Breast Cancer Research Predoctoral Fellowship_ \hfill 2010 – 2014  
Department of Defense, W81XWH-11-1-0088  
"Molecular Regulatory Network Dysregulation in Breast Cancer Cell Migration & Invasion"

_Graduate Research Fellowship_ \hfill 2009 – 2014  
National Science Foundation

_Momenta Presidential Fellowship_ \hfill 2009  
Massachusetts Institute of Technology

# Teaching Experience

_Guest Speaker_, Bioinformatics 202 \hfill 2023  
UCLA, Bioinformatics Interdepartmental Program

- Discussed our lab's research and related topics in bioinformatics.

_Guest Lecturer_, Fundamentals of Digital Imaging and Image Processing \hfill 2021, 2022  
UCLA, Molecular, Cell, and Developmental Biology M130

- Led discussion of a paper from the lab used as a project within the class

_Discussion Leader_, Ethics and Accountability in Biomedical Research \hfill 2021, 2024  
UCLA, Microbiology, Immunology, & Molecular Genetics

- Led discussion of various ethics case studies

<!-- Evaluations are available for file. -->

_Advisor_, Integrated and Interdisciplinary Undergraduate Research Program \hfill 2019 – 2023  
UCLA, Undergraduate Research Center

- Advise program participants on developing research, presentation, and professional skills

_Instructor_, Machine Learning & Data-Driven Modeling in Bioengineering \hfill 2018 – Present  
UCLA, Department of Bioengineering

- Designed and lead project-based course tailored to the background of students in the program

_Instructor_, Bioengineering Laboratory \hfill 2018 – Present  
UCLA, Department of Bioengineering

- Lead lab-based course introduction to laboratory work in bioengineering and basics of experimental design and analysis

_Mentor_, Bioengineering Capstone \hfill 2017, 2018, 2019, 2020  
UCLA, Department of Bioengineering

- Mentored three capstone teams for the bioengineering senior design course
- Poster competition winning team: 2018, 2019

_Guest Speaker_, Introduction to Bioengineering \hfill 2017, 2019, 2020, 2023  
UCLA, Department of Bioengineering

- Guest speaker to discuss research program and opportunities in bioengineering

_Faculty of the Citizen Science Program_ \hfill 2015 – 2016  
Bard College, Citizen Science Program, Annandale-on-Hudson, NY

- Led a short course introducing students to the natural sciences and scientific method

_Teaching Assistant_, Thermodynamics of Biomolecular Systems \hfill 2010  
MIT, Department of Biological Engineering, Cambridge, MA

# Conference & Invited Presentations (Last Five Years)

_Cytokine Based Drug Development Summit_, Invited Oral Presentation \hfill May 2024  
"New Cytokine Targeting Strategies Enabled by Multivalent Cis-Targeted Complexes."

_NIAID Systems Biology Consortium Webinar_, Invited Oral Presentation \hfill May 2024  
"Tensor Modeling of Clinical Outcomes in S. aureus Bacteremia."

_Tracer Precision Health Workshop_, Invited Oral Presentation \hfill April 2024  
"Mechanistic, integrative, and high-resolution dissection of single-cell studies with PARAFAC2."

_Cancer Systems Biology Program_, Invited Oral Presentation \hfill December 2023  
"Analysis and modeling of cancer drug responses using cell cycle phase-specific rate effects."

Systems Biology Consortium for Infectious Diseases, Invited Oral Presentation \hfill September 2023  
"Developing integrative signatures across omics, studies, and diseases with tensor-based analysis."

_UCLA Bioinformatics Retreat_, Invited Oral Presentation \hfill July 2023  
"Building the tensor learning universe."

_Antibodies & Complement_, Selected Oral Presentation \hfill June 2023  
"Cancer systems serology reveals active humoral immunity but disrupted Fc-elicited interactions."

_CSBC Annual Meeting_, Invited Oral Presentation \hfill March 2023  
"Phosphoproteomic Analysis of AXL Identifies YAP as a Key Regulator of Resistance."

_UCLA Musculoskeletal Devices & Tech. Development Group_, Invited Seminar \hfill September 2022  
"Uncovering immunologic mechanisms of MRSA persistence by tensor-mediated data integration."

_SIAM Conference on Mathematics of Data Science_, Invited Podium Presentation \hfill September 2022  
"Mechanistic and Data-Driven Dissection of Cell Communication Through Tensor Methods."

_American Assoc. for Cancer Research Annual Meeting_, Invited Podium Presentation \hfill April 2022  
"Systems approaches for identifying cell states and pathways modulating therapy response."

_Southern California Systems Biology Conference_, Invited Podium Presentation \hfill April 2022  
"Mechanistic and Data-Driven Dissection of Cell Communication Through Tensor Methods."

_Johns Hopkins Univ., Institute for Comp. Medicine_, Invited Seminar \hfill February 2022  
"Mechanistic and Data-Driven Dissection of Cell Communication Through Tensor Methods."

_Cellular \& Molecular Bioengineering_, Selected Oral Presentation \hfill January 2022  
"Rapid Prototyping of Multivalent And Multi-Specific Drugs To Overcome The Limited Selectivity Of IL-2 Toward Regulatory T Cells"

_Biomedical Engineering Society Annual Meeting_, Invited Podium Presentation \hfill October 2021  
"Tensor Factorization-Based Data Fusion Improves Predictions and Interpretation of MRSA Outcome."

_CSHL Systems Immunology_, Selected Oral Presentation \hfill April 2021  
"Developing a Mechanistic View of Mixed IgG Antibody Immune Effector Responses."

_University of Massachusetts, Mol & Cell Biol Program_, Invited Seminar \hfill March 2021  
"Mixture models of cell populations and signaling to understand heterogeneous drug response."

_International Conference on Biomolecular Engineering_, Selected Oral Presentation \hfill January 2021  
"Developing a Mechanistic View of Mixed IgG Antibody Immune Effector Responses."

_Vanderbilt University, QSBC Center_, Invited Center Seminar \hfill October 2020  
"Mixture models of cell populations and signaling to understand heterogeneous drug response."

_Buffalo Quantitative Systems Pharmacology Symposium_, Invited Speaker \hfill July 2020  
"Deeply profiling pharmacodynamic response with single cell dynamics."  
_Postponed due to COVID-19._

_Tufts University, Dept. of Bioengineering_, Invited Dept. Seminar \hfill March 2020  
"Linking Statistical and Mechanistic Models for Drug Development."  
_Postponed due to COVID-19._

_Univ. of Calif., Los Angeles, Immunogenetics Center_, Invited Speaker \hfill Jan 2020  
"Using models with incomplete information to study and engineer antibody effector response."

_Biomedical Engineering Society Annual Meeting_, Selected Oral Presentation \hfill October 2019  
"A Binding Model Predicts _In Vivo_ Effector Cell-Elicited Killing Across Multiple Disease Models."

_Xencor, Inc._, Invited Oral Presentation \hfill July 2019  
"Computational molecular models for immune engineering."

_Antibodies & Complement_, Selected Oral Presentation \hfill May 2019  
"A Multivalent Binding Model Predicts FcγR Regulation and Effector Cell-Elicited Killing."

_CSBC West Coast Meeting_, Selected Oral Presentation \hfill May 2019  
"Hidden Markov models on a tree as a general approach to single cell plasticity analysis."

# Research Supervision

*Postdoctoral Fellows*

- Catera Wilder, Ph.D. (Assistant Professor, UCSF) \hfill 2018 – 2022
- Song Yi Bae, Ph.D. (Postdoctoral Fellow, University of Minnesota) \hfill 2016 – 2019
- Edward Richards, Ph.D. (American Cancer Society Postdoctoral Fellowship) \hfill 2015 – 2020
- Annelien Zweemer, Ph.D. (Asst. Prof., Leiden University) \hfill 2014 – 2017

*Ph.D. Students*

- Michelle Loui \hfill 2022 – Present
    - SURF Fellowship, UCLA Graduate Division
- Andrew Ramirez \hfill 2021 – Present
    - NSF Graduate Research Fellowship
    - Cota Robles Fellowship
    - UCLA EDI Student Leadership Award
- Jackson Chin \hfill 2020 – Present
    - Best Poster Award, QC Bio Retreat, 2022
- Brian Orcutt-Jahns \hfill 2019 – Present
    - Best Poster Award, CSBC Junior Investigator Meeting
    - Best Poster Award, Cytokine Society
    - Best Presentation Award, Los Angeles Bioscience Ecosystem Summit
    - Outstanding Ph.D. Award, Department of Bioengineering
- Cyrillus Tan \hfill 2019 – Present
    - Dissertation Year Fellowship, UCLA Graduate Division
- Farnaz Mohammadi \hfill 2018 – 2023
    - Dissertation Year Fellowship, UCLA Graduate Division
- Marc Creixell \hfill 2018 – 2023
    - JCCC Fellowship

*Undergraduate Students*

- Jamie Stickelmaier \hfill 2021 – 2023
- Ethan Hung (Amgen Scholar, Berkeley) \hfill 2021 – Present
- Eva Hunter \hfill 2021 – 2022
- Hakan Alpay (Frontend Engineer, Facebook) \hfill 2021
- Luka Karginov (NCI CSBC Summer Scholar; Ph.D., Biological Engineering, MIT) \hfill 2020 – 2021
- Madeleine Murphy (Computational Biologist, Broad Institute of MIT \& Harvard) \hfill 2020 – 2022
- Aditya Sivakumar \hfill 2020 – 2021
- Eli Snyder (M.D., University of Hawaii) \hfill 2020 – 2021
- Peter Emmel \hfill 2019 – 2022
- Amanda Tsao (M.D., University of Southern California) \hfill 2019 – 2021
- JC Lagarde \hfill 2019 – 2022
- Sumedha Kanthamneni (Google) \hfill 2019 – 2022
- Heather Carmen Mercieca (Amgen Scholar) \hfill 2019
- Linnet Chang (Analyst, Accenture) \hfill 2018 – 2021
- Stephen Lees (Ph.D., Biomedical Engineering, UVA) \hfill 2018 – 2021
- Zoe Kim (Engineer, GaN Corporation) \hfill 2018 – 2020
- Micah Bryant (M.S., Mechanical Engineering, UCSD) \hfill 2018 – 2020
- Robby Theisen (Ph.D., Biomedical Engineering, University of Michigan) \hfill 2018 – 2020
- Alison Tran (Biosciences Account Manager, Thermo Fisher Scientific) \hfill 2018 – 2020
- Willie Wu (Software Engineer, Rivian) \hfill 2018 – 2019
- Katrina Warner (Amgen Scholar; Ph.D., Biomedical Sciences, Harvard) \hfill 2018
- Donya Khashayar (Transfer Student Summer Research Program) \hfill 2018
- Rui Yan (Cathy Bank Scholarship; Ph.D., ICME, Stanford) \hfill 2017 – 2019
- Ali Farhat (Rose Hills Foundation Scholar; M.D./Ph.D., U Illinois) \hfill 2017 – 2019
- Adam Weiner (Internet Research Initiative Award; Ph.D., Tri-Institute CompBio) \hfill 2017 – 2019
- Ning Guan (Ph.D., Systems Biology, Harvard) \hfill 2015 – 2017
- Ryan Robinett (NSF GRFP; Ph.D., Comp. Sci., U. Chicago) \hfill 2015 – 2017

# Service to the Profession

_Chalk Talk Mentor_, BME Underrepresented Needs In Technology & Engineering (UNITE) \hfill 2024

_Conference Organizer_, NIH NIAID \hfill 2024  
Systems Immunology Approaches in Transplantation Tolerance and Rejection (SIATTR)

_Ad Hoc Reviewer_, Genome Medicine \hfill 2024

_Ad Hoc Study Section_, NCI Human Tumor Atlas Network \hfill 2024  

_Ad Hoc Reviewer_, Metabolomics \hfill 2023

_Ad Hoc Reviewer_, Cancer Gene Therapy \hfill 2023

_Ad Hoc Reviewer (4x)_, Science Signaling \hfill 2020–2023

_Ad Hoc Reviewer (3x)_, Science Advances \hfill 2020–2023

_Track Chair_, Biomedical Engineering Society Annual Meeting \hfill 2023  
Computational \& Systems Biology

_Abstract Reviewer_, UC Systemwide Bioengineering Symposium \hfill 2023

_F31 Co-Sponsor_, University of Texas, Austin \hfill 2022  
Mollie Harrison (Advisor Stephanie Seidlits)

_Ad Hoc Reviewer_, Cancer Immunology Research \hfill 2022

_Co-Chair_, Resource & Data Sharing Working Group \hfill 2022 – Present  
Cancer Systems Biology Consortium

_Webmaster_, BME Underrepresented Needs In Technology & Engineering (UNITE) \hfill 2022 – Present

_Ad Hoc Reviewer_, iScience \hfill 2022

_Poster Judge_, Cellular \& Molecular Bioengineering Meeting \hfill 2022

[_Example U01 proposal_](https://www.niaid.nih.gov/grants-contracts/sample-applications), NIH National Institute of Allergy and Infectious Diseases \hfill 2021

_Reviewer_, Australia Medical Research Future Fund \hfill 2022  
Preventive and Public Health Research Initiative  
Optimising the Clinical Use of Immunoglobulins Grant Assessment Committee

_Local Organizing Committee_, Southern California Systems Biology Conference \hfill 2022

_Ad Hoc Reviewer_, Soft Matter \hfill 2022

_Ad Hoc Reviewer_, FEBS Letters \hfill 2021

_Co-Organizer_, BME UNITE Webinar Series \hfill 2021 – Present  
Showcase of current and future faculty candidates diverse and underrepresented in BME

_Abstract Reviewer_, Biomedical Engineering Society Annual Meeting \hfill 2021, 2022

_Financial Officer_, Association of Cancer Systems Biologists \hfill 2021 – Present

_Session Co-Chair_, Biomedical Engineering Society Annual Meeting \hfill 2020

_Volunteer Speed Interviewer_, Biomedical Engineering Society Annual Meeting \hfill 2020

_Volunteer Resume Reviewer_, Biomedical Engineering Society Annual Meeting \hfill 2020

_Member_, BME Underrepresented Needs In Technology & Engineering (UNITE) \hfill 2020 – Present  
Lead for Project 7: Coordinated graduate student recruiting

_Panelist_, Amgen Scholars Summer Science Series \hfill 2020

_Ad Hoc Reviewer_, PLOS Biology \hfill 2020, 2021

_Ad Hoc Reviewer_, Cancer Research \hfill 2020

_External Reviewer_, Ming Hsieh Institute, USC \hfill 2020

_Ad Hoc Reviewer_, Cell Systems \hfill 2020

_Ad Hoc Reviewer_, APL Bioengineering \hfill 2020

_Ad Hoc Reviewer_, Integrative Biology \hfill 2019

_Ad Hoc Reviewer_, Scientific Reports \hfill 2019

_Ad Hoc Reviewer_, PNAS \hfill 2019

_Ad Hoc Reviewer_, Current Opinion in Systems Biology \hfill 2019

_Co-Chair_, Association of Cancer Systems Biologists \hfill 2017 – 2021

_Ad Hoc Reviewer_, PLOS Computational Biology \hfill 2018

_Interviewee_, [Prescriber Magazine](http://www.prescriber.co.uk/wp-content/uploads/sites/23/2017/12/Systems-Biology-lsw.pdf) \hfill 2017

_Ad Hoc Reviewer_, WIREs Systems Biology and Medicine \hfill 2017

_Ad Hoc Remote Reviewer_, Irish Research Council \hfill 2017

_Ad Hoc Reviewer_, Cell Reports \hfill 2017, 2023

_Graduate Research Fellowship Program Review Panelist_, National Science Foundation \hfill 2016 – 2017

_Meeting Organizer & Member_, Association of Early Career Cancer Systems Biologists \hfill 2015 – 2016

_Ad Hoc Reviewer_, Biomedical Engineering Society Annual Meeting \hfill 2016

_Ad Hoc Reviewer_, Drug Discovery Today \hfill 2016

_Ad Hoc Reviewer_, Molecular Cell \hfill 2015

_Member_, Biomedical Engineering Society \hfill 2010 – Present

_Coordinator_, MIT Biological Engineering Graduate Student Board \hfill 2010 – 2013

_Ad Hoc Reviewer_, Oncogene \hfill 2013

_Ad Hoc Reviewer_, Nature \hfill 2013

_Member_, MIT Biological Engineering Retreat Organizing Committee \hfill 2010 – 2012

_Ad Hoc Reviewer_, J. Cell Biol. \hfill 2011 – 2012

# Service to UCLA

_Member_, Hiring Committee \hfill 2024  
Hispanic Serving Institution (HSI) STEM Faculty Director

_Member_, HSSEAS Strategic Planning Committee \hfill 2024  
Artificial intelligence focus area

_Chair_, Campus Response to the Climate Crisis Special Committee, Faculty Senate \hfill 2023–2024

_Ad Hoc Member_, Executive Board, Faculty Senate \hfill 2023–2024

_Faculty Speaker_, UCLA Life Sciences Webinar Series \hfill 2023  
"Digital Immune Twins: The Future of Healthcare?"

_Faculty Speaker_, UCLA Life Sciences Webinar Series \hfill 2023  
"Digital Immune Twins: The Future of Healthcare?"

_Poster Judge_, Undergraduate Research and Creativity Showcase \hfill 2023  
Science Dean's Prize

_Poster Judge_, Jonsson Cancer Center Annual Retreat \hfill 2023

_Reviewer_, Tau Beta Pi Chapter Excellence Scholarship \hfill 2021–2023

_Award Selection Committee_, Faculty Career Development Award \hfill 2022  
Office of Equity, Diversity and Inclusion

_Faculty Participant_, Coffee Chat Series, Computational & Systems Biology \hfill 2022

_Member_, Minors Committee, Computational & Systems Biology \hfill 2021–2023

_Panelist_, Graduate School Panel, Computational & Systems Biology \hfill 2021

_Faculty Representative_, Samueli Engineering Grad School Info Session \hfill 2020

_Faculty Representative_, Annual Biomedical Research Conference for Minority Students \hfill 2018, 2020

_Curriculum Advisory Committee_, Computational & Systems Biology \hfill 2020 – Present

_Written Qualifying Exam Evaluator_, Bioinformatics IDP \hfill 2020

_Mentor_, B.I.G. Summer \hfill 2020 – 2022

_Member_, SPUR "Life of a Faculty Member" Panel \hfill 2020

_Ad Hoc Member_, HSSEAS Faculty Executive Committee \hfill April 2020

_Application Reviewer_, Amgen Scholars Program \hfill 2020, 2022, 2023

_Reviewer_, Graduate Division’s Faculty Review Committee \hfill 2020

_Co-Organizer_, UCLA Systems Immunology Seminar Series \hfill 2019 – 2020

_Faculty Volunteer_, Society of Women Engineers Recruitment Dinner \hfill 2019, 2020

_Member_, HSSEAS SEASnet Review Committee \hfill 2019

_Faculty Advisor_, Tau Beta Pi \hfill 2017 – Present

_Faculty Volunteer_, Amgen Scholars Symposium \hfill 2018, 2019

_Member_, HSSEAS Awards Committee for Outstanding Student Awards \hfill 2018

_Faculty Speaker_, UCLA Engineering Alumni Reunion \hfill 2018

# Service to the Department

_Poster Judge_, Bioengineering Research Day \hfill 2024

_Panel Member_, Equity, Diversity, & Inclusion Panel \hfill 2024  
Biomedical Engineering Society

_Member_, Ad Hoc P&T Committee \hfill 2023

_Judge_, Biomedical Engineering Society BioHack \hfill 2022 – 2023

_Member_, Diversity, Equity and Inclusion Committee \hfill 2021 – Present

_Member_, Bioengineering Hiring Search Committee \hfill 2021 – 2022, 2023 – 2024

_Member_, Teaching Facility & Shared Equipment Committee \hfill 2021 – Present

_Member_, Strategic Planning Committee \hfill 2020

_Co-Chair_, Bioengineering and Computational Medicine Joint Hiring Search \hfill 2019 – 2020

_Field Chair_, Biosystem Science and Engineering \hfill 2019 – Present

_Graduate Admissions Committee Co-Chair_, Bioengineering \hfill 2019 – Present

_Undergraduate Curriculum Committee_, Bioengineering \hfill 2019 – Present

_Member_, Bioengineering Alumni Committee \hfill 2018 – Present

_Chair_, Department of Bioengineering Seminar Series \hfill 2018 – 2019

_Member_, Publicity Committee \hfill 2017 – 2018

# Patents/Disclosures

A.S. Meyer. "Methods of Identifying and Correcting Tumor Humoral Immune Dysregulation." U.S. patent application PCT/US24/32940, 2024.

Orcutt-Jahns, B., P.C. Emmel, A.S. Meyer. "Multi-specific engineered cytokines." U.S. patent application 63/428,921, 2022.

A.S. Meyer. "Altering cytokine specificity through binding valency." U.S. patent application PCT/US22/35711, 2022.

Miller, M.A., M.J. Oudin, A.S. Meyer, L.G. Griffith, F.B. Gertler, D.A. Lauffenburger. "Methods of Reducing Kinase Inhibitor Resistance." US patent application 14/690,001, 2015.

# Thesis Committee Membership

\begin{multicols}{2}

Daniel Bradbury, Bioengineering (Ph.D.)\\
Advisor: Daniel Kamei\\
2017–2020

Giovanni Valdez, Bioengineering (Ph.D.)\\
Advisor: Grace Xiao\\
2018–2021

Hiromi Miwa, Bioengineering (Ph.D.)\\
Advisor: Dino Di Carlo\\
2019–2022

Mark van Zee, Bioengineering (Ph.D.)\\
Advisor: Dino Di Carlo\\
2019–2022

Rob Dimatteo, Bioengineering (Ph.D.)\\
Advisor: Dino Di Carlo\\
2019–2021

Alexander Wickstrom, Bioengineering (M.S.)\\
Advisor: Jonathan Kao\\
2019

Hector E Muñoz, Bioengineering (Ph.D.)\\
Advisor: Dino Di Carlo\\
2019–2020

Wei-Chia Elizabeth Luo, Bioengineering (Ph.D.)\\
Advisor: Gerard Wong\\
2020–Present

Mohammadali Alidoost, Bioengineering (Ph.D.)\\
Advisor: Jennifer Wilson\\
2021–Present

Cameron S. Movassaghi, Chemistry (Ph.D.)\\
Advisor: Anne M. Andrews\\
2021–Present

Felis Doyeon Koo, Bioengineering (Ph.D.)\\
Advisor: Dino Di Carlo\\
2021–2023

Favour Esedebe, Bioinformatics (Ph.D.)\\
Advisor: Tom Graeber\\
2021–Present

Connor Razma, Bioinformatics (M.S.)\\
Advisor: Alexander Hoffmann\\
2022–2023

Mai Tran, Earth, Planetary \& Space Sci. (Ph.D.)\\
Advisor: William Newman\\
2022–2023

Nilay Shah, Computer Science (M.S.)\\
Advisor: Bolei Zhou\\
2022–2023

Rayo Suseno, Bioengineering (M.S.)\\
Advisor: Jennifer Wilson\\
2022–2023

Shawn Liu, Bioengineering (M.S.)\\
Advisor: Jennifer Wilson\\
2023–Present

Helen Huang, Bioinformatics (Ph.D.)\\
Advisor: Alexander Hoffman\\
2022–Present

Emily Bozich, Bioengineering (Ph.D.)\\
Advisor: Jennifer Wilson\\
2023–Present

Jingwen Sun, Chemistry \& Biochemistry (Ph.D.)\\
Advisor: Chong Liu\\
2023–Present

Seth Hilliard, Comp. \& Quant. Medicine (Ph.D.)\\
Advisor: Andrei Rodin (City of Hope)\\
2023

Frances Nicklen, Bioengineering (Ph.D.)\\
Advisor: Daniel Kamei\\
2023–Present

Michael Mellody, Bioengineering (Ph.D.)\\
Advisor: Dino Di Carlo\\
2023–Present

James Popoli, Bioengineering (Ph.D.)\\
Advisor: Andrea Kasko\\
2023–Present

Citra Soemardy, Bioengineering (Ph.D.)\\
Advisor: Dino Di Carlo\\
2023–Present

Shivani Kumar, Bioengineering (M.S.)\\
Advisor: Mireille Kamariza\\
2024

Alejandro Miron Jabalera, Bioengineering (Ph.D.)\\
Advisor: Tzung Hsiai\\
2024–Present

Ahmed Ali, Bioengineering (M.S.)\\
Advisor: Jennifer Wilson\\
2023–2024

\end{multicols}
