---
name: Aaron S. Meyer
url: http://asmlab.org
email: ameyer@ucla.edu
phone: 310-794-4821
addrOne: 4121G Engineering V
addrTwo: Los Angeles, CA 90095
---

# Education

_Ph.D._, Biological Engineering  \hfill April 2014  
Massachusetts Institute of Technology, Cambridge, MA  
Thesis: Quantitative approaches to understanding signaling regulation of 3D cell migration
 
_B.S._, Bioengineering, magna cum laude \hfill June 2009  
University of California, Los Angeles, CA

# Professional Experience

_Assistant Professor_ \hfill 2017 – Present  
Bioengineering Department, University of California, Los Angeles  
Bioinformatics Interdepartmental Graduate Program, University of California, Los Angeles  
Computational & Systems Biology Interdepartmental Program, University of California, Los Angeles

_Principal Investigator & Research Fellow_ \hfill 2014 – 2017  
Koch Cancer Institute, MIT, Cambridge, MA

_Graduate Researcher in the labs of Douglas Lauffenburger & Frank Gertler_ \hfill 2009 – 2014  
Department of Biological Engineering & Koch Cancer Institute, MIT, Cambridge, MA

_Undergraduate Researcher in the lab of Daniel Kamei_ \hfill 2006 – 2009  
Department of Bioengineering, University of California, Los Angeles, CA

# Submitted Publications Under Review/Revision

\leftskip 0.1in
\parindent -0.1in

VanDyke, D., M. Iglesias, J. Tomala, A. Young, J. Bridge, J. Perry, E. Gebara, A.R. Cross, L.S. Cheung, A.G. Dykema, **B. Orcutt-Jahns**, T. Henclová, J. Golias, J. Balolong, L.M. Tomasovic, D. Funda, **A.S. Meyer**, D.M. Pardoll, J. Hester, F. Issa, C.A. Hunter, M.S. Anderson, J.A. Bluestone, G. Raimondi, J.B. Spangler. Engineered human cytokine/antibody fusion proteins elicit targeted expansion of regulatory T cells and confer protection against autoimmune diseases." _Submitted._

\parskip 0.1in

**Orcutt-Jahns, B**, **P.C. Emmel**, **E.M. Snyder**, Cori Posner, Scott M. Carlson, **Aaron S. Meyer**. "Multivalency enhances the specificity of Fc-cytokine fusions." _Submitted, [bioRxiv preprint](https://www.biorxiv.org/content/10.1101/2021.07.03.451002v1)._

**Mohammadi, F.**, **S. Visagan**, S.M. Gross, **L. Karginov**, **JC Lagarde**, L.M. Heiser, **A.S. Meyer**. "A lineage tree-based hidden Markov model to quantify cellular heterogeneity and plasticity." _Submitted, [bioRxiv preprint](https://www.biorxiv.org/content/10.1101/2021.06.25.449922v1)._

S.M. Gross, **F. Mohammadi**, C. Sanchez-Aguila, P.J. Zhan, **A.S. Meyer**, L.M. Heiser. "Analysis and modeling of cancer drug responses using cell cycle phase-specific rate effects." _Submitted, [bioRxiv preprint](https://www.biorxiv.org/content/10.1101/2020.07.24.219907v2)._

# Refereed Publications

\leftskip 0.1in
\parindent -0.1in

**Creixell, M.**, Hyuna Kim, **F. Mohammadi**, S.R. Peyton, **A.S. Meyer**. "Systems approaches to uncovering the contribution of environment-mediated drug resistance." _Current Opinion in Solid State & Materials Science._ _Accepted._

\parskip 0.1in

**Creixell, M.**, **A.S. Meyer**. ["Dual data and motif clustering improves the modeling and interpretation of phosphoproteomic data."](https://doi.org/10.1016/j.crmeth.2022.100167) _Cell Reports Methods._ 2022 Feb 28; 2(2): 100167.

Majumder, A., S. Hosseinian, M.J. Stroud, E. Adhikari, J.J. Saller, D.M.A. Smith, D.G. Zhang, S. Agarwal, **M. Creixell**, B.S. Meyer, M.F. Kinose, K.S. Bowers, B. Fang, P.A. Stewart, E.A. Welsh, T.A. Boyle, **A.S. Meyer**, J.M. Koomen, E.B. Haura. ["Proteomic Characterization of AXL Kinase Inhibitors and Signaling Pathways."](https://mcr.aacrjournals.org/content/early/2022/01/12/1541-7786.MCR-21-0275.abstract) _Molecular Cancer Research._ 2022 Jan 7; 20(4): 542–555.

**Tan, Z.C.**, **B.T. Orcutt-Jahns**, **A.S. Meyer**. ["A quantitative view of strategies to engineer cell-selective ligand binding."](https://doi.org/10.1093/intbio/zyab019) _Integrative Biology._ 2021 Nov 23; 13(11): 269–282.

**Tan, Z.C.**, **A.S. Meyer**. ["A general model of multivalent binding with ligands of heterotypic subunits and multiple surface receptors."](https://doi.org/10.1016/j.mbs.2021.108714) _Mathematical Biosciences._ 2021 Dec; 342:108714.

**Tan, Z.C.**, **M.C. Murphy**, **H.S. Alpay**, **S.D. Taylor**, **A.S. Meyer**. ["Tensor-structured decomposition improves systems serology analysis."](https://doi.org/10.15252/msb.202110243) _Molecular Systems Biology._ 2021 Sept 6; 17:e10243.

**Farhat, A.M.**, **A.C. Weiner**, C. Posner, **Z.S. Kim**, **B. Orcutt-Jahns**, S.M. Carlson, **A.S. Meyer**. ["Modeling Cell-Specific Dynamics and Regulation of the Common Gamma Chain Cytokines."](https://doi.org/10.1016/j.celrep.2021.109044) _Cell Reports._ 2021 April 27; 35(4):109044.

**Bae, S.Y.**, **N. Guan**, **R. Yan**, **K. Warner**, **S.D. Taylor**, **A.S. Meyer**. ["Measurement and models accounting for cell death capture hidden variation in compound response."](https://www.nature.com/articles/s41419-020-2462-8) _Cell Death & Disease._, 2020 Apr 20; 255(11). 

Lee, C.-H., T. H. Kang, O. Godon, M. Watanabe, G. Delidakis, C.M. Gillis, D. Sterlin, D. Hardy, M. Cogné, L.E. Macdonald, A.J. Murphy, N. Tu, J. Lee, J.R. McDaniel, E. Makowski, Peter M. Tessier, **A.S. Meyer**, P. Bruhns, G. Georgiou. ["An engineered human Fc domain that behaves like a pH-toggle switch for ultra-long circulation persistence."](https://www.nature.com/articles/s41467-019-13108-2) _Nature Communications._, 2019 Nov 6; 10(1):5031.

**Meyer, A.S.**, L.M. Heiser. ["Systems biology approaches to measure and model phenotypic heterogeneity in cancer."](https://doi.org/10.1016/j.coisb.2019.09.002) _Current Opinion in Systems Biology._ 2019 Oct 4; 17: 35--40.

Situ, K., B.A. Chua, **S.Y. Bae**, **A.S. Meyer**, K. Morizono. ["Versatile targeting system for lentiviral vectors involving biotinylated targeting molecules."](https://doi.org/10.1016/j.virol.2018.09.017) _Virology._ 2018 Oct 2; 525: 170--181.

**Robinett, R.A.**, **N. Guan**, A. Lux, M. Biburger, F. Nimmerjahn, **A.S. Meyer**. ["Dissecting FcγR Regulation Through a Multivalent Binding Model."](https://doi.org/10.1016/j.cels.2018.05.018) _Cell Systems._ 2018 Jul 25; 6(7): 1--8.

Claas, A.M., L. Atta, S. Gordonov, **A.S. Meyer**, D.A. Lauffenburger. ["Systems Modeling Identifies Divergent Receptor Tyrosine Kinase Reprogramming to MAPK Pathway Inhibition."](https://link.springer.com/article/10.1007/s12195-018-0542-y) _Cellular and Molecular Bioengineering_ 2018 Jul 26; 1--19.

Muffat, J., Y. Li, A. Omer, A. Durbin, I. Bosch, G. Bakiasi, **E. Richards**, **A.S. Meyer**, L. Gehrke, R. Jaenisch. ["Human iPS-derived Glial Cells and Neural Progenitors Display Divergent Responses to Zika and Dengue Infections."](https://www.pnas.org/content/115/27/7117) _Proc. Natl. Acad. Sci. U.S.A._ 2018 Jun 18; 201719266.

Schwartz, A.D., L.E. Barney, L.E. Jansen, T.V. Nguyen, C.L. Hall, **A.S. Meyer**, S.R. Peyton. ["A Biomaterial Screening Approach to Reveal Microenvironmental Mechanisms of Drug Resistance."](http://pubs.rsc.org/en/content/articlelanding/2017/ib/c7ib00128b) _Integrative Biology._ 2017 Nov 14; 9: 912--924.

**Zweemer, A.J.M.**, C.B. French, J. Mesfin, S. Gordonov, **A.S. Meyer**, D.A. Lauffenburger. ["Apoptotic Cell Bodies Elicit Gas6-Mediated Migration Of AXL-Expressing Tumor Cells."](https://doi.org/10.1158/1541-7786.MCR-17-0012) _Molecular Cancer Research._ 2017 Sept 18; 15 (12): 1656--1666.

Archer, T.C., E.J. Fertig, S.J.C. Gosline, M. Hafner, S.K. Hughes, B.A. Joughin, **A.S. Meyer\footnote{Corresponding author.}**, S.P. Piccolo, A. Shajahan-Haq. ["Systems Approaches to Cancer Biology."](https://dx.doi.org/10.1158%2F0008-5472.CAN-16-1580) _Cancer Research._ 2016 Nov 18; 76 (23); 1--4.

**Manole, S.**, **E.J. Richards**, **A.S. Meyer**. ["JNK pathway activation modulates acquired resistance to EGFR/HER2 targeted therapies."](https://aacrjournals.org/cancerres/article/76/18/5219/615800/JNK-Pathway-Activation-Modulates-Acquired) _Cancer Research._ 2016 Sept 15; 76 (18): 5219--5228.

McConnell, R.E., J.E. Van Veen, M. Vidaki, A.V. Kwiatkowski, **A.S. Meyer**, D.A. Lauffenburger, F.B. Gertler. ["A Requirement for Filopodia Extension Towards Slit During Robo-Mediated Axon Repulsion."](http://jcb.rupress.org/content/213/2/261.full) _Journal of Cell Biology._ 2016 Apr 18; 213 (2): 261.

Miller, M.A., M.J. Oudin, R.J. Sullivan, D.T. Frederick, **A.S. Meyer**, S. Wang, H. Im, J. Tadros, L.G. Griffith, H. Lee, R. Weissleder, K.T. Flaherty, F.B. Gertler, D.A. Lauffenburger. ["Reduced proteolytic shedding of receptor tyrosine kinases is a post-translational mechanism of kinase inhibitor resistance."](http://cancerdiscovery.aacrjournals.org/content/early/2016/03/15/2159-8290.CD-15-0933) _Cancer Discovery._ 2016 Apr; 6:331-333.

Miller, M.A., M. Moss, G. Powell, R. Petrovich, L. Edwards, **A.S. Meyer**, L.G. Griffith, D.A. Lauffenburger. ["Targeting autocrine HB-EGF signaling with specific ADAM12 inhibition using recombinant ADAM12 prodomain."](http://www.ncbi.nlm.nih.gov/pubmed/26477568) _Scientific Reports._ 2015 Oct 19; 5:15150.

**Meyer\footnote{Co-corresponding authors.}, A.S.**, **A.J.M. Zweemer**, D.A. Lauffenburger\footnotemark[\value{footnote}]. ["The AXL receptor is a sensor of ligand spatial heterogeneity."](http://dx.doi.org/10.1016/j.cels.2015.06.002) _Cell Systems._ 2015 Nov 29; 1(1):25-36.

Riquelme, D.N., **A.S. Meyer**, M. Barzik, A. Keating, F.B. Gertler. ["Selectivity in subunit composition of Ena/VASP tetramers."](http://www.ncbi.nlm.nih.gov/pubmed/26221026) _Biosci. Rep._ 2015 Jul 28;35(5). pii: e00246.

**Meyer, A.S.**, M.A. Miller, F.B. Gertler, D.A. Lauffenburger. ["The receptor AXL diversifies EGFR signaling and limits the response to EGFR-targeted inhibitors in triple-negative breast cancer cells."](http://www.ncbi.nlm.nih.gov/pubmed/23921085) _Science Signaling._ 2013 Aug 6; 6(287):ra66.

Miller\footnote{Equally contributing authors.}, M.A., **A.S. Meyer**\footnotemark[\value{footnote}], M. Beste, Z. Lasisi, S. Reddy, K. Jeng, C.-H. Chen, J. Han, K. Isaacson, L.G. Griffith, D.A. Lauffenburger. ["ADAM-10 and -17 regulate endometriotic cell migration via concerted ligand and receptor shedding feedback on kinase signaling."](http://www.ncbi.nlm.nih.gov/pubmed/23674691) _Proc. Natl. Acad. Sci. U.S.A._ 2013 May 28; 110(22):E2074-83.

**Meyer, A.S.**, S.K. Hughes-Alford, J.E. Kay, A. Castillo, A. Wells, F.B. Gertler, D.A. Lauffenburger. ["2D protrusion but not motility predicts growth factor-induced cancer cell migration in 3D collagen."](http://www.ncbi.nlm.nih.gov/pubmed/22665521) _Journal of Cell Biology._ 2012 Jun 11; 197(6):721-9.

Kim, H.D., **A.S. Meyer**, J.P. Wagner, S.K. Alford, A. Wells, F.B. Gertler, D.A. Lauffenburger. ["Signaling network state predicts Twist-mediated effects on breast cell migration across diverse growth factor contexts."](http://www.ncbi.nlm.nih.gov/pubmed/21832255) _Mol. Cell. Proteomics._ 2011 Nov;10(11):M111.008433.

**Meyer, A.S.**, R.G. Condon, G. Keil, N. Jhaveri, Z. Liu, Y.-S. Tsao. ["Fluorinert, an oxygen carrier, improves cell culture performance in deep square 96-well plates by facilitating oxygen transfer."](http://www.ncbi.nlm.nih.gov/pubmed/21954223) _Biotechnol. Prog._ 2012 Jan; 28(1):171-8.

Mashayekhi, F., **A.S. Meyer**, S.A. Shiigi, V. Nguyen, D.T. Kamei. ["Concentration of mammalian genomic DNA using two-phase aqueous micellar systems."](http://www.ncbi.nlm.nih.gov/pubmed/19061237) _Biotechnol. Bioeng._ 2009 Apr 15; 102(6):1613-23.

\leftskip 0.0in
\parindent 0.0in

# Research Support & Awards

_COVID Relief Funds_, Vice Chancellor for Research Office \hfill 2022

_Milstein Abstract Award_, Cytokine Society \hfill 2021  
"Multivalency enhances the specificity of Fc-cytokine fusions"

_Northrop Grumman Excellence in Teaching Award_ \hfill 2021  

_Administrative Supplement to U01-CA215709_ \hfill 2020 – 2021  
"Mechanistic Autoencoders for Patient-Specific Phosphoproteomic Models"

_Grant_ \hfill 2020 – 2022  
Jayne Koskinas Ted Giovanis Foundation  
"Cell cycle-specific drug responses in breast cancer"

_American Cancer Society, Research Scholar Grant (co-I)_  \hfill 2020 – 2023  
"Tissue-engineered models of glioblastoma for evaluating treatment responses"

_NIH NIAID U01-AI148119_ \hfill 2019 – 2024  
Fc-Dependent Mechanisms of Antibody-Mediated Killing Consortium  
"Mapping the effector response space of antibody combinations"

_UCLA Faculty Career Development Award_ \hfill 2019 – 2020  

_UCLA Hellman Fellow_ \hfill 2019 – 2020  
"Engineering anti-tumor antibody combinations for more effective and less toxic therapies"

_Visterra, Inc. Research Agreement_ \hfill 2019 – 2021  
"IL-2 Receptor Binding Engineering"

_Administrative Supplement to U01-CA215709_ \hfill 2018 – 2019  
"Cell lineage analysis to quantify heterogeneous cell cycle responses of cancer cells"

_NIH NCI U01-CA215709_ \hfill 2017 – 2022  
Cancer Systems Biology Consortium  
"Precision Lung Cancer Therapy Design through Multiplexed Adapter Measurement"

_Fellowship Grant_ \hfill 2017 – 2019  
Terri Brodeur Breast Cancer Foundation  
"Decoding the Role of TAM Receptors _In Vivo_ Using More Specific and Potent Inhibitors"

_[Ten to Watch](http://www.amgenscholars.com/alumni/ten-to-watch)_, Amgen Scholars Foundation \hfill 2016

_AMIGOS Program Award_ \hfill 2016 – 2020  
Jayne Koskinas Ted Giovanis Foundation and Breast Cancer Research Foundation  
"Understanding the Role of Cell Plasticity in Mediating Drug Resistance"

_GPU Grant_ \hfill 2016  
NVIDIA Corporation  
"Parameterizing Stochastic Cell Signaling Pathways Through Variability Fitting"

_Frontier Research Program Initiator Award_ \hfill 2015  
Koch Institute for Integrative Cancer Research  
"Multiplexed Tools for Probing Chemokine Receptor Activation State in Breast Cancer"

_NIH Director's Early Independence Award, DP5-OD019815_  \hfill 2014 – 2019  
"Adapter-Layer RTK Signaling: Basic Understanding & Targeted Drug Resistance"  
**[Highlighted by the NIH director's office.](https://directorsblog.nih.gov/2014/10/28/creative-minds-tackling-chemotherapy-resistance/)**

_Siebel Scholar, Class of 2014_ \hfill 2013 

_Whitaker Fellowship_  \hfill 2013  
Massachusetts Institute of Technology

_Repligen Fellowship in Cancer Research_ \hfill 2012  
Koch Institute for Integrative Cancer Research

_Frontier Research Program Initiator Award_ \hfill 2011  
Koch Institute for Integrative Cancer Research  
"Global Growth Factor Reprogramming and Invasion By AXL Expression And Shedding In Breast Carcinoma"

_Breast Cancer Research Predoctoral Fellowship_ \hfill 2010 – 2014  
Department of Defense, W81XWH-11-1-0088  
"Molecular Regulatory Network Dysregulation in Breast Cancer Cell Migration & Invasion"

_Graduate Research Fellowship_ \hfill 2009 – 2014  
National Science Foundation

_Momenta Presidential Fellowship_ \hfill 2009  
Massachusetts Institute of Technology

# Teaching Experience

_Guest Lecturer_, Fundamentals of Digital Imaging and Image Processing \hfill 2021, 2022  
UCLA, Molecular, Cell, and Developmental Biology M130

- Led discussion of a paper from the lab used as a project within the class

_Discussion Leader_, Ethics and Accountability in Biomedical Research \hfill 2021  
UCLA, Microbiology, Immunology, & Molecular Genetics

- Led discussion of various ethics case studies

<!-- Evaluations are available for file. -->

_Advisor_, Integrated and Interdisciplinary Undergraduate Research Program \hfill 2019 – Present  
UCLA, Undergraduate Research Center

- Advise program participants on developing research, presentation, and professional skills

_Instructor_, Machine Learning & Data-Driven Modeling in Bioengineering \hfill 2018 – Present  
UCLA, Department of Bioengineering

- Designed and lead project-based course tailored to the background of students in the program

_Instructor_, Bioengineering Laboratory \hfill 2018 – Present  
UCLA, Department of Bioengineering

- Lead lab-based course introduction to laboratory work in bioengineering and basics of experimental design and analysis

_Mentor_, Bioengineering Capstone \hfill 2017, 2018, 2019, 2020  
UCLA, Department of Bioengineering

- Mentored three capstone teams for the bioengineering senior design course
- Poster competition winning team: 2018, 2019

_Guest Speaker_, Introduction to Bioengineering \hfill 2017, 2019, 2020  
UCLA, Department of Bioengineering

- Guest speaker to discuss research program and opportunities in bioengineering

_Faculty of the Citizen Science Program_ \hfill 2015 – 2016  
Bard College, Citizen Science Program, Annandale-on-Hudson, NY

- Led a short course introducing students to the natural sciences and scientific method

_Teaching Assistant_, Thermodynamics of Biomolecular Systems \hfill 2010  
MIT, Department of Biological Engineering, Cambridge, MA

# Conference & Invited Presentations (Last Five Years)

_SIAM Conference on Mathematics of Data Science_, Invited Podium Presentation \hfill September 2022  
"Mechanistic and Data-Driven Dissection of Cell Communication Through Tensor Methods."

_American Assoc. for Cancer Research Annual Meeting_, Invited Podium Presentation \hfill April 2022  
"Systems approaches for identifying cell states and pathways modulating therapy response."

_Southern California Systems Biology Conference_, Invited Podium Presentation \hfill April 2022  
"Mechanistic and Data-Driven Dissection of Cell Communication Through Tensor Methods."

_Johns Hopkins Univ., Institute for Comp. Medicine_, Invited Seminar \hfill February 2022  
"Mechanistic and Data-Driven Dissection of Cell Communication Through Tensor Methods."

_Cellular \& Molecular Bioengineering_, Selected Oral Presentation \hfill January 2022  
"Rapid Prototyping of Multivalent And Multi-Specific Drugs To Overcome The Limited Selectivity Of IL-2 Toward Regulatory T Cells"

_Biomedical Engineering Society Annual Meeting_, Invited Podium Presentation \hfill October 2021  
"Tensor Factorization-Based Data Fusion Improves Predictions and Interpretation of MRSA Outcome."

_CSHL Systems Immunology_, Selected Oral Presentation \hfill April 2021  
"Developing a Mechanistic View of Mixed IgG Antibody Immune Effector Responses."

_University of Massachusetts, Mol & Cell Biol Program_, Invited Seminar \hfill March 2021  
"Mixture models of cell populations and signaling to understand heterogeneous drug response."

_International Conference on Biomolecular Engineering_, Selected Oral Presentation \hfill January 2021  
"Developing a Mechanistic View of Mixed IgG Antibody Immune Effector Responses."

_Vanderbilt University, QSBC Center_, Invited Center Seminar \hfill October 2020  
"Mixture models of cell populations and signaling to understand heterogeneous drug response."

_Buffalo Quantitative Systems Pharmacology Symposium_, Invited Speaker \hfill July 2020  
"Deeply profiling pharmacodynamic response with single cell dynamics."  
_Postponed due to COVID-19._

_Tufts University, Dept. of Bioengineering_, Invited Dept. Seminar \hfill March 2020  
"Linking Statistical and Mechanistic Models for Drug Development."  
_Postponed due to COVID-19._

_Univ. of Calif., Los Angeles, Immunogenetics Center_, Invited Speaker \hfill Jan 2020  
"Using models with incomplete information to study and engineer antibody effector response."

_Biomedical Engineering Society Annual Meeting_, Selected Oral Presentation \hfill October 2019  
"A Binding Model Predicts _In Vivo_ Effector Cell-Elicited Killing Across Multiple Disease Models."

_Xencor, Inc._, Invited Oral Presentation \hfill July 2019  
"Computational molecular models for immune engineering."

_Antibodies & Complement_, Selected Oral Presentation \hfill May 2019  
"A Multivalent Binding Model Predicts FcγR Regulation and Effector Cell-Elicited Killing."

_CSBC West Coast Meeting_, Selected Oral Presentation \hfill May 2019  
"Hidden Markov models on a tree as a general approach to single cell plasticity analysis."

_Oregon Health & Science Univ., Dept. of Biomedical Engineering_, Invited Dept. Seminar \hfill March 2019  
"Systems approaches to mapping and targeting immune system communication."

_Univ. of Calif., Los Angeles, MSTP Tutorial Series_, Invited Speaker \hfill October 2018  
"Systems approaches to mapping and targeting immune system communication."

_Univ. of Illinois at Urbana-Champaign, Dept. of Bioengineering_, Invited Dept. Seminar \hfill Sept 2018  
"High-dimensional analysis to map and manipulate immune receptor-ligand families."

_Systems Biology of Human Disease_, Selected Oral Presentation \hfill June 2018  
"Dissecting FcγR Regulation Through a Multivalent Binding Model."

_Univ. of Calif., Riverside, Department of Bioengineering_, Invited Departmental Seminar \hfill April 2018  
"Dissecting FcγR Regulation Through a Multivalent Binding Model."

_Univ. of Bergen, Centre for Cancer Biomarkers_, Invited Speaker \hfill March 2018  
"Engineering more precise and potent TAM-targeted therapies."

_Univ. of Calif., Los Angeles, Dept. of Bioengineering_, Invited Departmental Seminar \hfill October 2017  
"Dissecting FcγR Regulation Through a Multivalent Binding Model."

_Univ. of Calif., Los Angeles, Broad Stem Cell Research Center_, Invited Speaker \hfill October 2017  
"Engineering more precise and potent TAM-targeted therapies."

_Momenta Pharmaceuticals_, Invited Oral Presentation \hfill April 2017  
"Dissecting FcγR Regulation Through a Multivalent Binding Model."

# Research Supervision

*Postdoctoral Fellows*

- Catera Wilder, Ph.D. (Assistant Professor, UCSF) \hfill 2018 – 2022
- Song Yi Bae, Ph.D. (Postdoctoral Fellow, University of Minnesota) \hfill 2016 – 2019
- Edward Richards, Ph.D. (American Cancer Society Postdoctoral Fellowship) \hfill 2015 – 2020
- Annelien Zweemer, Ph.D. (Asst. Prof., Leiden University) \hfill 2014 – 2017

*Ph.D. Students*

- Andrew Ramirez (NSF GRFP; Cota Robles) \hfill 2021 – Present
- Jackson Chin \hfill 2020 – Present
- Brian Orcutt-Jahns \hfill 2019 – Present
- Cyrillus Tan \hfill 2019 – Present
- Farnaz Mohammadi \hfill 2018 – Present
- Marc Creixell \hfill 2018 – Present

*Undergraduate Students*

- Hakan Alpay (Frontend Engineer, Facebook) \hfill 2021
- Luka Karginov (NCI CSBC Summer Scholar; Ph.D., Biological Engineering, MIT) \hfill 2020 – 2021
- Madeleine Murphy (Computational Biologist, Broad Institute of MIT \& Harvard) \hfill 2020 – Present
- Aditya Sivakumar \hfill 2020 – 2021
- Eli Snyder (Associate Consultant, Putnam Associates) \hfill 2020 – 2021
- Amanda Tsao (M.D., University of Southern California) \hfill 2019 – 2021
- JC Lagarde \hfill 2019 – Present
- Sumedha Kanthamneni \hfill 2019 – Present
- Heather Carmen Mercieca (Amgen Scholar) \hfill 2019
- Linnet Chang (Analyst, Accenture) \hfill 2018 – 2021
- Stephen Lees (Ph.D., Biomedical Engineering, UVA) \hfill 2018 – 2021
- Zoe Kim (Engineer, GaN Corporation) \hfill 2018 – 2020
- Micah Bryant (M.S., Mechanical Engineering, UCSD) \hfill 2018 – 2020
- Robby Theisen (Ph.D., Biomedical Engineering, University of Michigan) \hfill 2018 – 2020
- Alison Tran (Biosciences Account Manager, Thermo Fisher Scientific) \hfill 2018 – 2020
- Willie Wu (Software Engineer, Rivian) \hfill 2018 – 2019
- Katrina Warner (Amgen Scholar; Ph.D., Biomedical Sciences, Harvard) \hfill 2018
- Donya Khashayar (Transfer Student Summer Research Program) \hfill 2018
- Rui Yan (Cathy Bank Scholarship; Ph.D., ICME, Stanford) \hfill 2017 – 2019
- Ali Farhat (Rose Hills Foundation Scholar; M.D./Ph.D., U Illinois) \hfill 2017 – 2019
- Adam Weiner (Internet Research Initiative Award; Ph.D., Tri-Institute CompBio) \hfill 2017 – 2019
- Ning Guan (Ph.D., Systems Biology, Harvard) \hfill 2015 – 2017
- Ryan Robinett (NSF GRFP; Ph.D., Comp. Sci., U. Chicago) \hfill 2015 – 2017

# Service to the Profession

_Ad Hoc Reviewer_, Cancer Immunology Research \hfill 2022

_Co-Chair_, Resource & Data Sharing Working Group \hfill 2022 – Present  
Cancer Systems Biology Consortium

_Ad Hoc Reviewer_, iScience \hfill 2022

_Poster Judge_, Cellular \& Molecular Bioengineering Meeting \hfill 2022

[_Example U01 proposal_](https://www.niaid.nih.gov/grants-contracts/sample-applications), NIH National Institute of Allergy and Infectious Diseases \hfill 2021

_Reviewer_, Australia Medical Research Future Fund \hfill 2022  
Preventive and Public Health Research Initiative  
Optimising the Clinical Use of Immunoglobulins Grant Assessment Committee

_Local Organizing Committee_, Southern California Systems Biology Conference \hfill 2022

_Ad Hoc Reviewer_, Soft Matter \hfill 2022

_Ad Hoc Reviewer_, Science Signaling \hfill 2021

_Ad Hoc Reviewer_, FEBS Letters \hfill 2021

_Abstract Reviewer_, Biomedical Engineering Society Annual Meeting \hfill 2021

_Session Co-Chair_, Biomedical Engineering Society Annual Meeting \hfill 2020

_Volunteer Speed Interviewer_, Biomedical Engineering Society Annual Meeting \hfill 2020

_Volunteer Resume Reviewer_, Biomedical Engineering Society Annual Meeting \hfill 2020

_Ad Hoc Reviewer_, Science Signaling \hfill 2020

_Ad Hoc Reviewer (2x)_, Science Advances \hfill 2020

_Panelist_, Amgen Scholars Summer Science Series \hfill 2020

_Ad Hoc Reviewer_, PLOS Biology \hfill 2020, 2021

_Ad Hoc Reviewer_, Cancer Research \hfill 2020

_External Reviewer_, Ming Hsieh Institute, USC \hfill 2020

_Ad Hoc Reviewer_, Cell Systems \hfill 2020

_Ad Hoc Reviewer_, APL Bioengineering \hfill 2020

_Ad Hoc Reviewer_, Integrative Biology \hfill 2019

_Ad Hoc Reviewer_, Scientific Reports \hfill 2019

_Ad Hoc Reviewer_, PNAS \hfill 2019

_Ad Hoc Reviewer_, Current Opinion in Systems Biology \hfill 2019

_Co-Chair_, Association of Early Career Cancer Systems Biologists \hfill 2017 – Present

_Ad Hoc Reviewer_, PLOS Computational Biology \hfill 2018

_Interviewee_, [Prescriber Magazine](http://www.prescriber.co.uk/wp-content/uploads/sites/23/2017/12/Systems-Biology-lsw.pdf) \hfill 2017

_Ad Hoc Reviewer_, WIREs Systems Biology and Medicine \hfill 2017

_Ad Hoc Remote Reviewer_, Irish Research Council \hfill 2017

_Ad Hoc Reviewer_, Cell Reports \hfill 2017

_Graduate Research Fellowship Program Review Panelist_, National Science Foundation \hfill 2016 – 2017

_Meeting Organizer & Member_, Association of Early Career Cancer Systems Biologists \hfill 2015 – 2016

_Ad Hoc Reviewer_, Biomedical Engineering Society Annual Meeting \hfill 2016

_Ad Hoc Reviewer_, Drug Discovery Today \hfill 2016

_Ad Hoc Reviewer_, Molecular Cell \hfill 2015

_Member_, Biomedical Engineering Society \hfill 2010 – Present

_Coordinator_, MIT Biological Engineering Graduate Student Board \hfill 2010 – 2013

_Ad Hoc Reviewer_, Oncogene \hfill 2013

_Ad Hoc Reviewer_, Nature \hfill 2013

_Member_, MIT Biological Engineering Retreat Organizing Committee \hfill 2010 – 2012

_Ad Hoc Reviewer_, J. Cell Biol. \hfill 2011 – 2012

# Service to UCLA

_Reviewer_, Tau Beta Pi Chapter Excellence Scholarship \hfill 2021, 2022

_Award Selection Committee_, Faculty Career Development Award \hfill 2022  
Office of Equity, Diversity and Inclusion

_Member_, Minors Committee, Computational & Systems Biology \hfill 2021, 2022

_Panelist_, Graduate School Panel, Computational & Systems Biology \hfill 2021

_Faculty Representative_, Samueli Engineering Grad School Info Session \hfill 2020

_Faculty Representative_, Annual Biomedical Research Conference for Minority Students \hfill 2018, 2020

_Curriculum Advisory Committee_, Computational & Systems Biology \hfill 2020 – Present

_Written Qualifying Exam Evaluator_, Bioinformatics IDP \hfill 2020

_Mentor_, B.I.G. Summer \hfill 2020

_Member_, SPUR "Life of a Faculty Member" Panel \hfill 2020

_Ad Hoc Member_, HSSEAS Faculty Executive Committee \hfill April 2020

_Application Reviewer_, Amgen Scholars Program \hfill 2020, 2022

_Reviewer_, Graduate Division’s Faculty Review Committee \hfill 2020

_Co-Organizer_, UCLA Systems Immunology Seminar Series \hfill 2019 – 2020

_Faculty Volunteer_, Society of Women Engineers Recruitment Dinner \hfill 2019, 2020

_Member_, HSSEAS SEASnet Review Committee \hfill 2019

_Faculty Advisor_, Tau Beta Pi \hfill 2017 – Present

_Faculty Volunteer_, Amgen Scholars Symposium \hfill 2018, 2019

_Member_, HSSEAS Awards Committee for Outstanding Student Awards \hfill 2018

_Faculty Speaker_, UCLA Engineering Alumni Reunion \hfill 2018

# Service to the Department

_Judge_, Biomedical Engineering Society BioHack \hfill 2022

_Member_, Bioengineering Hiring Search Committee \hfill 2021 – 2022

_Member_, Teaching Facility & Shared Equipment Committee \hfill 2021 – Present

_Member_, Strategic Planning Committee \hfill 2020

_Co-Chair_, Bioengineering and Computational Medicine Joint Hiring Search \hfill 2019 – 2020

_Field Chair_, Biosystem Science and Engineering \hfill 2019 – Present

_Graduate Admissions Committee Co-Chair_, Bioengineering \hfill 2019 – Present

_Undergraduate Curriculum Committee_, Bioengineering \hfill 2019 – Present

_Member_, Bioengineering Alumni Committee \hfill 2018 – Present

_Chair_, Department of Bioengineering Seminar Series \hfill 2018 – 2019

_Member_, Publicity Committee \hfill 2017 – 2018

# Patents/Disclosures

**A.S. Meyer**. "Altering cytokine specificity through binding valency." U.S. provisional patent application 63/216,718, 2021.

Bae, S.Y., **A.S. Meyer**. "Small Molecule Competitive Inhibitors Of Phosphatidylserine-TAMR Ligand Interaction." Disclosure filed, 2018.

Richards, E.J., S. Manole, **A.S. Meyer**. "Modulating JNK activation to impede lung & breast cancer RTK inhibitor bypass resistance." Disclosure filed, 2016.

Miller, M.A., M.J. Oudin, **A.S. Meyer**, L.G. Griffith, F.B. Gertler, D.A. Lauffenburger. "Methods of Reducing Kinase Inhibitor Resistance." US patent application 14/690,001, 2015.

# Thesis Committee Membership

\begin{multicols}{2}

Daniel Bradbury, Bioengineering\\
Advisor: Daniel Kamei

Giovanni Valdez, Bioengineering\\
Advisor: Grace Xiao

Hiromi Miwa, Bioengineering\\
Advisor: Dino Di Carlo

Mark van Zee, Bioengineering\\
Advisor: Dino Di Carlo

Rob Dimatteo, Bioengineering\\
Advisor: Dino Di Carlo

Alexander Wickstrom, Bioengineering (M.S.)\\
Advisor: Jonathan Kao

Hector E Muñoz, Bioengineering\\
Advisor: Dino Di Carlo

Wei-Chia Elizabeth Luo, Bioengineering\\
Advisor: Gerard Wong

Mohammadali Alidoost, Bioengineering\\
Advisor: Jennifer Wilson

Cameron S. Movassaghi, Chemistry\\
Advisor: Anne M. Andrews

Felis Doyeon Koo, Bioengineering\\
Advisor: Dino Di Carlo

Favour Esedebe, Bioinformatics\\
Advisor: Tom Graeber

Connor Razma, Bioinformatics (M.S.)\\
Advisor: Alexander Hoffmann

Mai Tran, Earth, Planetary \& Space Sciences\\
Advisor: William Newman

Nilay Shah, Computer Science (M.S.)\\
Advisor: Bolei Zhou

\end{multicols}
